main.go

```diff
@@ -20,12 +20,13 @@ func main() {
 		"disable option to skip pretty printing")
 	encrypt_flag := flag.Bool("enc", false, "encrypt a given challenge")
 	decrypt_flag := flag.Bool("dec", false, "decrypt a given challenge")
+	detect_level_flag := flag.Bool("detect-level", false, "detect a level from a given hash")
```

Comment:
@mrshu Shouldn't be here `detect a level from a given hash and path to home directory` ?